src/App/App.tsx

```diff
@@ -15,6 +19,19 @@ import AppContent from './AppContent';
 export default function App() {
     const isShuttingDown = useShuttingDown();
     const isSettingUp = useAppStateStore((s) => s.isSettingUp);
+    const setError = useAppStateStore((s) => s.setError);
+    const setIsWebglNotSupported = useUIStore((s) => s.setIsWebglNotSupported);
+    const { t } = useTranslation('common', { useSuspense: false });
+
+    useEffect(() => {
+        if (!window.WebGL2RenderingContext && !window.WebGLRenderingContext) {
+            Sentry.captureMessage('WebGL not supported by the browser', { extra: { userAgent: navigator.userAgent } });
```

Comment:
> and send info to the sentry

I think if we've identified WebGL is not supported, we've prevented the context error, there's no need to send any info to sentry. It's just noise at that point. Maybe best to log it, so it's recorded somewhere local to the instance.

Reply:
Error won't be sent to sentry since I prevented from animation init. Decided do send messages to sentry to check what browsers users use when it happens

Reply:
So I can maybe improve this